json-endpoints/jetpack/json-api-jetpack-endpoints.php

```diff
@@ -1193,3 +1193,39 @@
 	'example_request' => 'https://public-api.wordpress.com/rest/v1/sites/example.wordpress.org/users/create'
 
 ) );
+
+require_once( $json_jetpack_endpoints_dir . 'class.jetpack-json-api-jps-woocommerce-connect-endpoint.php' );
+
+// POST /sites/%s/jps/woo-connect
+new Jetpack_JSON_API_JPS_WooCommerce_Connect_Endpoint( array(
+	'description'    => 'Attempts to connect the WooCommerce plugin for this site to WooCommerce.com.',
+	'group'          => '__do_not_document',
+	'method'         => 'POST',
+	'path'           => '/sites/%s/jps/woo-connect',
+	'stat'           => 'jps:woo-connect',
```

Comment:
it doesn't need to be declared as 'secure' this side too?

Reply:
I don't believe so. The endpoints for creating and connecting users in this same file aren't declared as being forced to secure even though they are on WPCOM.